.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file must be ignored. Please remove it first using `git rm -r .vscode/` (you can save a copy outside from the project in case you don't want to lose it), add `.vscode/` to `.gitignore` and commit that change. After that, you can restore de .vscode folder and it will be ignored by git

Reply:
All ready done it!